pnpm-lock.yaml

```diff
@@ -378,6 +378,7 @@ importers:
 
   packages/histoire-controls:
```

Comment:
Revert this file

Reply:
Done here: https://github.com/histoire-dev/histoire/pull/310/commits/67c61e29a3b9e75c71f4a06c2ae1e29712259a99